### PR TITLE
Add wrong syntax to trigger PHPCS error

### PR DESCRIPTION
**Pull request by mbtamuli**
Signed-off-by: Mriyam Tamuli <mbtamuli@gmail.com>

### CODE REVIEW

**Review by chandrapatel — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 3 errors 

***

**hashes-api**-scanning skipped

File: wp-locale-news.php [hunk 1]
```diff
@@ -49,10 +49,10 @@ function wpln_load_plugin_textdomain() {
 
 	}
 
-	$mofile = sprintf( '%1$s-%2$s.mo', 'wp-locale-news', $locale );
+	$mofile = sprintf('%1$s-%2$s.mo', 'wp-locale-news', $locale);
 
 	// Look in wp-content/languages/plugins/wp-locale-news.
-	$global_mofile = WP_LANG_DIR . '/plugins/wp-locale-news/' . $mofile;
+	$global_mofile = WP_LANG_DIR.'/plugins/wp-locale-news/' . $mofile;
```
Comment (chandrapatel) [inline wp-locale-news.php:55]: :no_entry_sign: **Error**: Concat operator must be surrounded by a single space

File: wp-locale-news.php [hunk 2]
```diff
@@ -49,10 +49,10 @@ function wpln_load_plugin_textdomain() {
 
 	}
 
-	$mofile = sprintf( '%1$s-%2$s.mo', 'wp-locale-news', $locale );
+	$mofile = sprintf('%1$s-%2$s.mo', 'wp-locale-news', $locale);
```
Comment (chandrapatel) [inline wp-locale-news.php:52]: :no_entry_sign: **Error**: Expected 1 spaces before closing bracket; 0 found

Comment (chandrapatel) [inline wp-locale-news.php:52]: :no_entry_sign: **Error**: Expected 1 spaces after opening bracket; 0 found

**Comment by rahul286** [comment]
@mbtamuli did we use @chandrapatel token for automated review or is this real human review?

**Comment by mbtamuli** [comment]
@rahul286 Correct. This is just for testing.

**Pull request closed without merging**